#!/usr/bin/python3


def raise_exception_msg(message=""):
    """this is to Raise a NameError exception with a message."""
    raise NameError(message)
